mappings/net/minecraft/text/Style.mapping

```diff
@@ -150,8 +150,36 @@ CLASS net/minecraft/class_2583 net/minecraft/text/Style
 		ARG 3 underlined
 		ARG 4 strikethrough
 		ARG 5 obfuscated
+	METHOD method_54210 with (Lnet/minecraft/class_2583;Ljava/lang/Object;Ljava/lang/Object;)Lnet/minecraft/class_2583;
+		ARG 0 newStyle
+		ARG 1 oldAttribute
+		ARG 2 newAttribute
 	CLASS class_2584 Serializer
 		COMMENT A JSON serializer for {@link Style}.
```

Comment:
It's no longer specific to JSON as it contains codecs instead. Would it be useful to rename to class to `Codecs` too, and would that make this comment in turn redundant?
```suggestion
		COMMENT The codecs for {@link Style}.
```